Packs/PaloAltoNetworks_Threat_Vault/Integrations/ThreatVaultv2/ThreatVaultv2.py

```diff
@@ -275,6 +354,20 @@ def resp_to_hr(response: dict, type_: str, expanded: bool = False) -> dict:
                 "CVE": response.get("cve"),
             }
 
+        case "rtdns" | "dns" | "spywarec2":
```

Comment:
these get used quite often. If we use them in conditions, I think they're worth constants (as well as `spyware` etc)